hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -3,6 +3,7 @@
 import com.google.inject.Guice;
 import com.google.inject.Inject;
 import com.google.inject.Injector;
```

Comment:
Let's try to keep our imports in alphabetical order. Understood that this isn't consistent throughout the whole project.